dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -480,6 +480,10 @@ boolean catchesLinkageError(String sourceClassName) {
       ClassGen classGen = new ClassGen(sourceJavaClass);
 
       for (Method method : sourceJavaClass.getMethods()) {
+        if (method.getCode() == null) {
```

Comment:
what sort of methods are these? abstract? native? If so, I'm surprised we haven't hit this before.

Reply:
I wrote my investigation in #1352 , which was caused by an invalid class file.

Reply:
I'd bet this is a BCEL error. The doc for Method.getCode even says "Code attribute of method, if any" so I don't think returning null here is a surprise.